libgpiod-monitor/main.cpp

```diff
@@ -0,0 +1,211 @@
+#include <atomic>
+#include <iostream>
+#include <chrono>
+#include <future>
+#include <gpiod.h>
+#include <ctype.h>
+#include <errno.h>
+#include <getopt.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string>
+#include <thread>
+#include <unistd.h>
+
+
+// constants
```

Comment:
Please transfer whole line comment from `//` to `/* ... */` for the coding style.  Include the following line comments.